src/main/java/controllers/MainUIController.java

```diff
@@ -12,15 +13,34 @@
  */
 public class MainUIController {
 
+	@FXML private TableViewController tableViewController;
+	@FXML private DataController dataController;
+	@FXML private AnalysisController analysisController;
+//	@FXML private VisualizationController visualizationController;
```

Comment:
Ok, in het vervolg is het waarschijnlijk wel mooier, als jij dan al in ieder geval de class aanmaakt, want uitgecommente code wil ik altijd gewoon weg hebben, want als je dat moet onthouden, kan je gewoon git gebruiken.